feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.ts

```diff
@@ -54,6 +93,39 @@ export class ConfiguratorAttributeDropDownComponent
     useFeatureStyles('productConfiguratorAttributeTypesV2');
   }
 
+  /**
+   * Extracts the relevant value prices from the price supplements
+   * and stores them within the component. Returns a boolean indicating
+   * whether there were any changes.
+   * @param config current config
+   * @returns {true}, only if at least one value price changed
+   */
+  protected checkedForValuePriceChanges(
```

Comment:
Could you add missing tests? 
![image](https://github.com/SAP/spartacus/assets/61147963/6a8a96ae-08f1-4088-9dc4-8a597b89d447)